### PR TITLE
Release version 0.13.0

### PR DESCRIPTION
**Pull request by adericbourg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.13.0] - 2018-23-07
+
+### Added
+- Elasticsearch driver (supports metadata only)
```
Comment (iksaif) [inline CHANGELOG.md]: Add link to ELASTICSEARCH_DESIGN.md

File: ELASTICSEARCH_DESIGN.md [hunk 2]
```diff
@@ -1,51 +1,65 @@
...
-This document describes data model for storing and retrieving metadata and timeseries data to/from ElasticSearch
+This document describes data model for storing and retrieving metadata and timeseries data to/from
+Elasticsearch.
 
 
 # Indices
 
 ## Metadata
 
+Metric names are split per component: from `p0` to `p63` (default value).
```
Comment (iksaif) [inline ELASTICSEARCH_DESIGN.md]: Maybe add some text about `name` and `config`  + the fact that they are stored in a set of time based indices.
Maybe add the default index mapping here, we should be pretty self-explanatory.

File: ELASTICSEARCH_DESIGN.md [hunk 3]
```diff
@@ -1,51 +1,65 @@
...
 ```
-
-# criteo.rtb.*.*.*.*.*.*.*.??_???.web-rtb*
-GET biggraphite/_search
+# criteo.{rtb,cas}.*.*.*.*.*.*.*.??_???.web-rtb*
+GET biggraphite-metrics*/_search
 {
    "_source": ["name", "uuid", "created_on", "read_on", "updated_on", "config"],
-  "size": "9999",
+  "size": "10000",
```
Comment (iksaif) [inline ELASTICSEARCH_DESIGN.md]: Ths line can be removed

File: CHANGELOG.md (hunk 1 above)
Comment (adericbourg) [reply CHANGELOG.md]: Done.

File: ELASTICSEARCH_DESIGN.md (hunk 2 above)
Comment (adericbourg) [reply ELASTICSEARCH_DESIGN.md]: Done.

File: ELASTICSEARCH_DESIGN.md (hunk 3 above)
Comment (adericbourg) [reply ELASTICSEARCH_DESIGN.md]: Done.

**Pull request merged**